relay/kusama/src/lib.rs

```diff
@@ -1849,6 +1851,23 @@ pub mod migrations {
 		}
 	}
 
+	/// Cancel all ongoing auctions.
+	///
+	/// Any leases that come into existance, after coretime was launched will not be served. Yet,
+	/// any still ongoing auctions must be cancelled.
+	///
+	/// Safety: After coretime is launched, there are no auctions anymore. So if this forgotten to
+	/// be removed after the runtime upgrade, running this again on the next one is harmless.
+	pub struct CancelAuctions;
+	impl OnRuntimeUpgrade for CancelAuctions {
+		fn on_runtime_upgrade() -> Weight {
+			if let Err(err) = Auctions::cancel_auction(frame_system::RawOrigin::Root.into()) {
```

Comment:
What about the scheduled auctions? I see there is currently one auction scheduled. Generally not sure if we always scheduled them manually?

Reply:
Good point!

Reply:
Ok, where are they handled?

Reply:
They should be under the Scheduler pallet's `Agenda`. We will just have to kill any items that are auctions.